src/EditCell.tsx

```diff
@@ -53,10 +55,15 @@ export default function EditCell<R, SR>({
   });
 
   useEffect(() => {
-    if (!commitOnOutsideClick) return;
-
-    function onWindowCaptureMouseDown() {
-      frameRequestRef.current = requestAnimationFrame(commitOnOutsideMouseDown);
+    function onWindowCaptureMouseDown(event: MouseEvent) {
+      if (event.target instanceof Element) {
+        outsideClickedElementRef.current = event.target;
```

Comment:
Idea is to keep track of the clicked element and if it outside the grid then do not steal focus